src/styles/header/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+  box-sizing: border-box;
```

Comment:
You can move `box-sizing: border-box;` to a global style file to apply it to all elements. It's a common practice to do so, because it makes sizing elements much more predictable.